Tests/ConnectionFactoryTest.php

```diff
@@ -7,59 +7,19 @@
 use Doctrine\DBAL\Configuration;
 use Doctrine\DBAL\Connection;
 use Doctrine\DBAL\Driver;
-use Doctrine\DBAL\Driver\API\ExceptionConverter;
-use Doctrine\DBAL\Driver\Exception as InternalDriverException;
-use Doctrine\DBAL\Exception as DBALException;
-use Doctrine\DBAL\Exception\DriverException;
-use Doctrine\DBAL\Platforms\AbstractPlatform;
-use Doctrine\DBAL\Platforms\MySQLPlatform;
-use Doctrine\DBAL\Platforms\SqlitePlatform;
-use Doctrine\DBAL\Schema\AbstractSchemaManager;
 use Doctrine\Deprecations\PHPUnit\VerifyDeprecations;
-use Exception;
 
 use function array_intersect_key;
-use function class_exists;
-use function defined;
-
-// Compatibility with DBAL < 3
-class_exists(\Doctrine\DBAL\Platforms\MySqlPlatform::class);
 
 class ConnectionFactoryTest extends TestCase
 {
     use VerifyDeprecations;
 
-    public function testContainer(): void
```

Comment:
I don't understand what this test actually tests. That exception throwing logic all seems DBAL related.